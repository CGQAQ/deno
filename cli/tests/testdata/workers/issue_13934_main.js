// main.js
new Worker(
  new URL("./worker1.js", import.meta.url).href,
  { type: "module" },
);
