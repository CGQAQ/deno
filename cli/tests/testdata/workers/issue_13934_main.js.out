Worker 1
Current Web worker don't have deno namespace but child wants it.
